src/amr/data/field/initializers/field_user_initializer.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef _PHARE_AMR_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_
+#define _PHARE_AMR_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_
+
+#include "initializer/data_provider.hpp"
```

Comment:
here you should probably declare a bit more given what's being used below

it might look like 

```
#include "core/utilities/span.hpp"
#include "initializer/data_provider.hpp"

#include <tuple>
#include <memory>
```

Reply:
https://github.com/include-what-you-use/include-what-you-use ?

Reply:
[yes](https://gist.github.com/PhilipDeegan/d8879297e72d26f5512ddf5663a3133e)?